src/site/content/en/fast/content-delivery-networks/index.md

```diff
@@ -0,0 +1,284 @@
+
+
+*  **Elimination of head-of-line blocking**
+
+    HTTP/2 introduced multiplexing, a feature that allows a single connection to be used to transmit multiple streams of data simultaneously. However, with HTTP/2, a single dropped packet blocks all streams on a connection (a phenomena known as a head-of-line blocking). With HTTP/3, a dropped packet only blocks a single stream. This improvement is largely the result of HTTP/3 using [UDP](https://en.wikipedia.org/wiki/User_Datagram_Protocol) rather than [TCP](https://en.wikipedia.org/wiki/Transmission_Control_Protocol). This makes HTTP/3 particularly useful for data transfer over congested or lossy networks.
+
+<figure class="w-figure">
+  <img src="./cdn4.png" alt="Diagram showing the differences in data transmission between HTTP/1, HTTP/2, and HTTP/3" class="w-screenshot">
+</figure>
+
+*  **Reduced connection setup time**
+
+    HTTP/3 uses TLS 1.3 and therefore shares its performance benefits: establishing a new connection only requires a single round-trip and resuming an existing connection does not require any roundtrips.
+
+<figure class="w-figure">
+  <img src="./cdn5.png" alt="Comparison of connection resumption between TLS 1.2 and TLS 1.3" class="w-screenshot">
+</figure>
+
+HTTP/3 will have the biggest impact on users on poor network connections: not only because HTTP/3 handles packet loss better than its predecessors, but also because the absolute time savings resulting from a 0-RTT or 1-RTT connection setup will be greater on networks with high latency.
+
+
+### Image optimization
+
+CDN image optimization services typically focus on image optimizations that can be applied automatically in order to reduce image transfer size. For example: stripping [EXIF](https://en.wikipedia.org/wiki/Exif) data, applying lossless compression, and converting images to newer file formats (for example, WebP). Images make up ~50% of the transfer bytes on the median web page, so optimizing images can significantly reduce page size.
+
+
+### Minification
+
+Minification removes unnecessary characters from JavaScript, CSS, and HTML. It's preferable to do minification at the origin server, rather than the CDN. Site owners have more context about the code to be minified and therefore can often use more aggressive minification techniques than those employed by CDNs. However, if minifying code at the origin is not an option, minification by the CDN is a good alternative.
+
+
+## Conclusion
+
+*   **Use a CDN:** CDNs deliver resources quickly, reduce load on the origin server, and are helpful for dealing with traffic spikes.
```

Comment:
On my first readthrough I didn't see much about the idea that "CDNs reduce load on the origin server" and I think it's a good one (because it might save someone money). You might want to go back and see if there's anywhere you can expand on this idea (even if it's only a sentence, and the sentence is very direct, e.g. "a CDN may save you money"). On the other hand you might already cover it well and my reading comprehension just wasn't that thorough.